portal/src/graphql/adminapi/AuditLogScreen.tsx

```diff
@@ -37,13 +43,37 @@ function CommandBarDropdownWrapper(props: ICommandBarItemProps) {
   return <CommandBarDropdown {...dropdownProps} />;
 }
 
+function RefreshButton(props: ICommandBarItemProps) {
+  const tooltipStyle: Partial<ITooltipHostStyles> = {
+    root: { display: "inline-block" },
+  };
+  const tooltipId = useId("refreshTooltip");
+  const tooltipCalloutProps = { gapSpace: 0 };
+  const tooltipcontent =
```

Comment:
Obtain the locale with `useContext(Context)` where `Context` is imported from `@oursky/react-messageformat`.